pkg/server/server.go

```diff
@@ -204,6 +205,12 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// separating scoped cluster client
+	scopedClusterKubeClient, err := kubernetes.NewForConfig(kcpclienthelper.NewClusterConfig(c.GenericConfig.LoopbackClientConfig))
```

Comment:
what is a scoped cluster client?

Reply:
So we have config's cluster Kube client (https://github.com/kcp-dev/kcp/blob/6e99317ea3a6a9fe10722f8c7fdb1dd50883c461/pkg/server/config.go#L187) but we can't change it right away to accept `Kubernetes.Interface` till informers (https://github.com/kcp-dev/kcp/blob/6e99317ea3a6a9fe10722f8c7fdb1dd50883c461/pkg/server/config.go#L192) are scoped. That's why I created `scopedClusterKubeClient` for now, but it would be removed when we scoped informers.